README.md

```diff
@@ -3,9 +3,8 @@
 This repository contains a template that can be used to seed a repository for a
 new Google open source project.
 
-See [go/releasing](http://go/releasing) (available externally at
-https://opensource.google/documentation/reference/releasing) for more information about
-releasing a new Google open source project.
+See [Go Releasing](https://opensource.google/documentation/reference/releasing) 
```

Comment:
The opensource.google copy is a few years out of date.  We shouldn't be pointing Googlers at it.